src/libs/remove-diff-signs.js

```diff
@@ -0,0 +1,44 @@
+/* Lasciate ogne speranza, voi ch'intrate. */
```

Comment:
This is a good time to fix the spelling in this comment.

It should be _ch'**e**ntrate_.

https://en.wikiquote.org/wiki/Inferno_(Dante)#Canto_III:_The_Gate_of_Hell

Reply:
There's some confusion online, some say `ch'e…` and some say `ch'i…`

This is the most reputable source I found so far and it uses `ch'i…`:

http://etcweb.princeton.edu/cgi-bin/dante/campuscgi/mpb/GetCantoSection.pl?INP_POEM=Inf&INP_SECT=3&INP_START=1&INP_LEN=200&LANG=1&submit=Go

Reply:
Oh well, I guess this 1350 text from Bocconi University might have to take precedence:

![](http://amshistorica.unibo.it/archivio/000170/000001/000015.jpg)

Source: http://amshistorica.unibo.it/170

Reply:
Also: https://en.wikiquote.org/wiki/Talk:Dante_Alighieri#Ogne_or_Ogni_and_Ch.27intrate_or_Chi.27entrate